aii-opennebula/src/main/perl/opennebula.pm

```diff
@@ -63,17 +65,21 @@ sub make_one
         $main::this_app->info ("Detected configfile RPC section: [$rpc]");
     };
     my $port = $config->{$rpc}->{port} || 2633;
-    my $host = $config->{$rpc}->{host} || "localhost";
+    my $host = $config->{$rpc}->{host};
+    my $url = $config->{$rpc}->{url} || "http://localhost:2633/RPC2";
```

Comment:
the defaults (inlc port and user) should be readonlys, like `Readonly my ONE_DEFAULT_URL => 'http://localhost:2633/RPC2';`